packages/apps/src/Endpoints/index.tsx

```diff
@@ -255,7 +272,7 @@ function Endpoints ({ className = '', offset, onClose }: Props): React.ReactElem
             <div className='endpointCustomWrapper'>
               <Input
                 className='endpointCustom'
-                isError={!isUrlValid}
+                isError={!isUrlValid || getApiType(apiUrl).type === 'json-rpc'}
```

Comment:
This should actually not show when light is selected at all or it should fallback to the previous value that was/is there -

```js
{getApiType(apiUrl).type === 'json-rpc' && (
  <div className='endpointCustomWrapper'>...
)}
```

The reasoning is that we now create error inputs instead of just checking what the user has.